lib/Backends/OpenCL/kernels.cl

```diff
@@ -15,6 +15,30 @@ typedef struct {
   cl_uint64_t c; // Number of channels
 } ShapeNHWC;
 
+#if defined(cl_khr_int32_base_atomics)
```

Comment:
can you do some indentation for readability?

Reply:
Indented the atomicAdd function. 
And preprocessor directives are usually not indented, or?